Add wellbeing resources and links (#71)
DFE-Digital/improve-school-workload-and-wellbeing: content/staff-wellbeing/academy-trust-workload-agreement.md

      </p>
      <p>
        Staff at Ascent can expect:
        <ul>
          <li>
            a fair and reasonable workload
          </li>
          <li>
            high quality training and professional development opportunities that meet the needs of individual members of staff
          </li>
        </ul>
      </p>
    </div>
  </div>
</div>

## Background from Carolyn Morgan, CEO

{inset-text}

It is clear from national surveys and Ascent Trust’s own fact finding, that excessive workload is a major obstacle to schools across the country being able to successfully recruit and retain the best teachers and support staff.  

We developed our academy trust workload agreement to support better recruitment and retention of teachers and support staff. 

{/inset-text}

## Ensuring fair and reasonable workload 

How the Ascent Trust ensures fair and reasonable workload:  

* the executive leadership team’s commitment to reducing work hours (above and beyond 1265) for all teachers and a Trust review of TA contractual hours and duties  

> * the appraisal of all staff should include a review of workload with a supporting action in order to ensure employers exercise their duty of care to employees with regard to workload, including heads of academy 

Author: ```suggestion
* we ensure that the appraisal of all staff includes a workload review.

* we take supporting actions to ensure employers exercise their duty of care to employees regarding workload, including heads of academy
```

* a total review of the Educational Health Care Plan (EHCP) reports and meetings, with Ascent (rather than individual LA) providing the lead on systems and procedures 

* a working party to review curriculum planning systems across the trust with the intention of reducing workload 

> * ensuring that before any new trust policies, reports or initiatives are introduced, Ascent completes a workload impact assessment to ensure there is no negative effect on workload 

Author: ```suggestion
* we ensure that Ascent completes a workload impact assessment before any new trust policies, reports or initiatives are introduced
```

---
title: Academy trust workload agreement 

> layout: /markdown_resource.html.erb

Author: I think I've made those changes and also grabbed some of the behaviour section?
